gtfs-realtime/proto/gtfs-realtime.proto

```diff
@@ -503,11 +521,15 @@ message VehiclePosition {
     // This message/field is still experimental, and subject to change. It may be formally adopted in the future.
     optional OccupancyStatus occupancy_status = 3 [default = NO_DATA_AVAILABLE];
 
+    // The passenger crowd level for this given carriage, in this vehicle.
+    // This message/field is still experimental, and subject to change. It may be formally adopted in the future.
+    optional CrowdLevel crowd_level = 4 [default = NO_DATA_AVAILABLE];
+
     // Occupancy percentage for this given carriage, in this vehicle.
     // Follows the same rules as "VehiclePosition.occupancy_percentage"
     // -1 in case data is not available for this given carriage (as protobuf defaults to 0 otherwise)
     // This message/field is still experimental, and subject to change. It may be formally adopted in the future.
-    optional int32 occupancy_percentage = 4 [default = -1];
+    optional int32 occupancy_percentage = 5 [default = -1];
```

Comment:
The .proto field value assignments are permanent and can't change after being added. So `occupancy_percentage` and `carriage_sequence` need to keep the same values, and `crowd_level` will need to have the value `6`.